.github/workflows/pr-coverage-report.yml

```
@@ -0,0 +1,65 @@
+          path: coverage/lcov-report/
+
+      - name: Upload coverage summaries
+        uses: actions/upload-artifact@v4
+        with:
+          name: jest-coverage-summaries
+          path: |
+            base-coverage-summary.json
+            pr-coverage-summary.json
+
+      - name: Update PR with Coverage Section
+        uses: ./.github/actions/javascript/updateCoverageSection
```

actionlint (1.7.7)

file "dist/index.js" does not exist in "/home/jailuser/git/.github/actions/javascript/updateCoverageSection". it is specified at "main" key in "runs" section in "Update PR with Test Coverage Section" action

(action)